OasisDEX/oasis-borrow
# Ajna mining rewards

# [Ajna mining rewards](https://app.shortcut.com/oazo-apps/story/8562/fe-ajna-lm-reward-claiming)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- updated designs
- added transaction handling (not tested yet)
- added data source
- fixed issue with vault api call using protocol with capital first letter
  
## How to test 🧪
  <Please explain how to test your changes>

- test whether designs are in line with figma (there will be minor differences due to agreement to not use magic numbers)
- check content of rewards page when connected and not connected (when connected banner in card should be displayed, if not connected banner should be hidden and connect button in page header should be visible)
- ajna token rewards card for now doesn't have banner because is not supported yet
- transaction handling is not testable due to lack of contract
